index.html

```diff
@@ -105,157 +87,107 @@ <h3 class="heading">Professional Art Printing Data</h3>
                   or sign-ups required. has been the industry's standard
                 </p>
                 <div class="skill-portfolio">
-                  <span class="badge"
-                    >HTML</span>
-                  <span class="badge"
-                    >Bootstrap</span
-                  >
-                  <span class="badge"
-                    >Ruby</span>
+                  <span class="badge-1">HTML</span>
+                  <span class="badge-1">Bootstrap</span>
+                  <span class="badge-1">Ruby</span>
                 </div>
```

Comment:
- [ ] Kindly make sure this list of technologies looks like the design. And please try to use different images for different projects.

<p align="center">&nbsp;
 
| Figma Design |   Your Project |
| ---------- | ----------- |
 | ![Template 2-Main](https://user-images.githubusercontent.com/12524453/177880175-a28e72ef-d0da-4459-906a-39589cae4948.png) | ![FireShot Capture 155 - Portfolio - localhost](https://user-images.githubusercontent.com/12524453/177880206-cb200df5-447a-4e13-bdb1-7b03944ec7f2.png) |
</p>